### PR TITLE
WebAPI: Update Method pages (part 43)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Updates definition lists.

### CODE REVIEW

File: files/en-us/web/api/filesystemdirectoryhandle/getdirectoryhandle/index.md [hunk 1]
```diff
@@ -33,9 +33,10 @@ var FileSystemDirectoryHandle = FileSystemDirectoryHandle.getDirectoryHandle();
   - : An optional object containing options for the retrieved subdirectory. Options are as
     follows:
 
-    - `create`: A {{jsxref('Boolean')}}. Default `false`. When
-      set to `true` if the directory is not found, one with the specified
-      name will be created and returned.
+    - `create`
+      - : A {{jsxref('Boolean')}}. Default `false`. When
```
Comment (teoli2003) [inline files/en-us/web/api/filesystemdirectoryhandle/getdirectoryhandle/index.md:37]:
```suggestion
      - : A boolean value, which defaults to `false`. When
```

**Comment by OnkarRuikar** [comment]
wait I am still working on it  :O

File: files/en-us/web/api/periodicsyncmanager/register/index.md [hunk 2]
```diff
@@ -21,19 +21,20 @@ resolves when the registration completes.
 ## Syntax
 
 ```js
-var register = PeriodicSyncManager.register(tag, BackgroundSyncOptions);
+register(tag, backgroundSyncOptions)
```
Comment (teoli2003) [inline files/en-us/web/api/periodicsyncmanager/register/index.md:24]: Let's use the simpler `options`:
```suggestion
register(tag, options)
```

File: files/en-us/web/api/periodicsyncmanager/register/index.md [hunk 3]
```diff
@@ -21,19 +21,20 @@ resolves when the registration completes.
...
 ```
 
 ### Parameters
 
 - `tag`
   - : A unique {{jsxref('String')}} identifier
-- _BackgroundSyncOptions_ {{optional_inline}}
+- `backgroundSyncOptions` {{optional_inline}}
```
Comment (teoli2003) [inline files/en-us/web/api/periodicsyncmanager/register/index.md:31]:
```suggestion
- `options` {{optional_inline}}
```

**Comment by teoli2003** [comment]
I made a few comments, as this PR is still in draft.

**Comment by OnkarRuikar** [comment]
@teoli2003  done with the changes. 
These are some of the last definition list fixes. Hopefully the next PR will complete everything.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/css/@charset/index.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ As there are several ways to define the character encoding of a style sheet, the
 
 where:
 
-- _charset_
+- `charset`
```
[the remark this answers is not in the document]
Comment (OnkarRuikar) [reply files/en-us/web/css/@charset/index.md:39]: ok

**Comment by OnkarRuikar** [comment]
I've reverted the changes from 11 CSS pages.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**